rb/spec/integration/selenium/webdriver/driver_spec.rb

```diff
@@ -299,6 +299,22 @@ def save_screenshot_and_assert(path)
           end.to raise_error(Selenium::WebDriver::Error::ScriptTimeoutError)
         end
       end
+
+      describe 'window' do
```

Comment:
Please move it to `./rb/spec/integration/selenium/webdriver/window_spec.rb`

Reply:
By moving it to the .switch_to API, wouldn't the test be located in:
`spec/integration/selenium/webdriver/target_locator_spec.rb`

Reply:
You're right, that would be a better place.